.github/workflows/lint.yml

```diff
@@ -25,6 +25,12 @@ jobs:
           version: v2.0.1
           args: --timeout 10m
           github-token: ${{ secrets.github_token }}
+      - name: Install golangci-lint
+        run: go install github.com/golangci/golangci-lint/v2/cmd/golangci-lint@v2.0.1
+        if: env.GIT_DIFF
+      - name: Run golangci-lint in multiplexer module
+        working-directory: multiplexer
+        run: golangci-lint run --timeout 10m
```

Comment:
why do we need separate steps for this? I would've expected the golangci-lint step above to run it on the whole repo because no directory is specified.